### PR TITLE
Timeline

### PR DESCRIPTION
**Pull request by tgomezzzz**
This PR implements the timeline in the Interests tab, as well as fixes formatting issues and makes the portfolio look nicer. The timeline is meant to show when I was interested in various topics, and moves across the tab as images representing each topic fade in. Each image can then be clicked for more information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Google Student Training in Engineering Program
 
+This repo contains Tommy's portfolio and STEP projects!
+
 This repo contains the projects you'll work on for the first half of STEP.
```
Comment (vidhyal) [inline README.md]: Please remove. Also, update the title accordingly.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,160 @@
...
+        <button class="tabbutton" onclick="displayTab(event, 'Interests')">My Interests</button>
+        <button class="tabbutton" onclick="displayTab(event, 'Bucket List')">My Bucket List</button>
+        <button class="tabbutton" onclick="displayTab(event, 'Gallery')">Gallery</button>
+      </div>
+      
+      <div id="About Me" class="tabinfo">
+        <div class="aboutme-header">
+          <h3> About Me
+          <a href="http://www.linkedin.com/in/tomgom"><img src="images/linkedin-logo.png" style="width: 3%; height: 3%; padding-left: 10px;"/></a>
+          <a href="http://www.github.com/tgomezzzz"><img src="images/github-logo.jpg" style="width: 3%; height: 3%; padding-left: 10px;"/></a>
```
Comment (vidhyal) [inline portfolio/src/main/webapp/index.html]: nit: another indent.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,17 +12,105 @@
...
+    buttonContainer.style.borderBottomRightRadius = "0";
+
+    var tabInfo = document.getElementsByClassName("tabinfo");
+    for (var i = 0; i < tabInfo.length; i++) {
+        tabInfo[i].style.display = "none";
+    }
+
+    var tabButtons = document.getElementsByClassName("tabbutton");
+    for (var i = 0; i < tabButtons.length; i++) {
+        tabButtons[i].className = tabButtons[i].className.replace(" active", "");
```
Comment (vidhyal) [inline portfolio/src/main/webapp/script.js:32]: Is the space here intended?

Comment (tgomezzzz) [reply portfolio/src/main/webapp/script.js:32]: Yes, this is intended.

**Review by vidhyal — APPROVED** [review]

**Pull request merged**